maci64_link.sh

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env bash
```

Comment:
nit: can the file be named something more descriptive? e.g. maci64_deterministic_link.sh ?